index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
Would the word "restrictions" or "constraints" help resolve the issue with the word "filters"?

"`data` may constrain which payment handlers can be used by specifying required capabilities. Only those payment handlers that have indicated their support for the capabilities specified in `data` match. If `data` provides no such constraints then any payment handler that supports the payment method identifier matches."